src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -878,6 +879,15 @@ public static void setup(@Nonnull SlimefunPlugin plugin) {
         false, false, "caveman", 50, new PotionEffect(PotionEffectType.FAST_DIGGING, 800, 2))
         .register(plugin);
 
+        new Talisman(SlimefunItems.TALISMAN_WISE,
+        new ItemStack[] { SlimefunItems.MAGIC_LUMP_3, null, SlimefunItems.MAGIC_LUMP_3, SlimefunItems.MAGICAL_GLASS, SlimefunItems.TALISMAN_MAGICIAN, SlimefunItems.FILLED_FLASK_OF_KNOWLEDGE, SlimefunItems.MAGIC_LUMP_3, null, SlimefunItems.MAGIC_LUMP_3},
```

Comment:
I think replacing the two currently empty slots with magical glass and requiring two filled flasks in the middle would be a good improvement.
We don't wanna make it too cheap ;)

Reply:
Sure